luigi/contrib/mongodb.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Big Datext Inc
```

Comment:
To be honest I have no idea what should go in there. I typically wrote the name of my current institution.